inscriptions/ISic000483.xml

```diff
@@ -186,23 +192,36 @@
       <desc>Add Description Here</desc>
     </graphic>
   </surface>
-
     -->
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AR">
+                <ab>
+                    <lb n="1"/><name>Furiae</name> <g ref="#interpunct">·</g> <name>Sa
+                    <lb n="2" break="no"/>biniae</name> <g ref="#interpunct">·</g> <name>Tran
+                    <lb n="3" break="no"/>quillinae</name>
+                    <lb n="4"/><expan><abbr>aug</abbr><ex>ustae</ex></expan> <g ref="#interpunct">·</g> coniugi 
+                    <lb n="5"/><expan><abbr><supplied reason="lost">i</supplied>mp</abbr><ex>eratoris</ex></expan> <g ref="#interpunct">·</g> <name><expan><abbr>M</abbr></expan><ex>arci</ex></name> <g ref="#interpunct">·</g> <name><expan><abbr>A</abbr><supplied reason="lost"><ex>ntoni</ex></supplied></expan></name>
```

Comment:
not: `<name><expan><abbr>M</abbr></expan><ex>arci</ex></name>`
but: `<name><expan><abbr>M</abbr><ex>arci</ex></expan></name>`
And there's no need for abbreviation marking on Antoni, just the supplied?